### DISCUSSION TITLE
Milvus 2.3.x standalone docker installation steps doesnt work

### DISCUSSION THREAD

**Discussion by rohanhodarkar** [milvus-io/milvus#30474] [Q&A and General discussion] [answered]
The steps mentioned for Milvus docker standalone installation here https://milvus.io/docs/install_standalone-docker.md doesn't work.

404 - Not found for https://raw.githubusercontent.com/milvus-io/milvus/master/scripts/standalone_embed.sh in first step.

**Comment by yhmo** [comment]
I think it is a typo of milvus document. We will update it later.

**Reply by rohanhodarkar** [reply]
Thank you @yhmo , Can you send me the corrected one if handy. Need it for installation.

**Reply by yhmo** [reply]
1. Download this yaml to a local folder
`wget https://github.com/milvus-io/milvus/releases/download/v2.3.7/milvus-standalone-docker-compose.yml -O docker-compose.yml`

2. In the local folder, use this command to deploy a local standalone server
`docker-compose up -d`

**Comment by yhmo** [comment, marked as answer]
Now the  https://raw.githubusercontent.com/milvus-io/milvus/master/scripts/standalone_embed.sh has been uploaded, you can download it to start a standalone.
